h2/src/main/org/h2/util/MathUtils.java

```diff
@@ -310,4 +322,12 @@ public static int secureRandomInt(int lowerThan) {
         return getSecureRandom().nextInt(lowerThan);
     }
 
+    private static Random getRandom() {
+        try {
+            Class.forName("java.util.concurrent.ThreadLocalRandom");
```

Comment:
Looks like a bad idea to do `Class.forName` each time.

Reply:
Ok, I'll think to make it better.

Reply:
Have a look at how LocalDateTimeUtils caches the result